package.json

```diff
@@ -28,7 +28,7 @@
     "should": "8.2.2"
   },
   "scripts": {
-    "test": "mocha --reporter spec --timeout 15000",
+    "test": "mocha --reporter spec --bail --timeout 15000",
```

Comment:
The bail option should really be restricted to tests that depend on each other and hit the real GitHub API for example.
For the rest, it's important to see what fails and what does not. This is because sometimes a test that fails can easily be explained because another more elementary test fails. And now we are using immutable structure, we don't care about order of tests, and cleaning up for most of the testing.

Can we add a specific script to run local tests only and remove `--bail` from it ?